backend/tests/controllers/test_executions/test_test_executions.py

```diff
@@ -291,10 +291,12 @@ def test_updates_test_execution(
     assert test_execution.ci_link == "http://ci_link/"
     assert test_execution.c3_link == "http://c3_link/"
     assert test_execution.status == TestExecutionStatus.PASSED
-    assert test_execution.review_decision == [
+    assert len(test_execution.review_decision) == 2
+    for review_decision in [
         TestExecutionReviewDecision.APPROVED_FAULTY_HARDWARE.name,
         TestExecutionReviewDecision.APPROVED_INCONSISTENT_TEST.name,
-    ]
+    ]:
+        assert review_decision in test_execution.review_decision
```

Comment:
while this is fine, a more pythonic way would be to just compare sets. If you define a set say `review_decision` then you can just compare it via `review_decision == test_execution.review_decision`. If you define it as a list you could just call `set()` on it

Reply:
I updated this. However, I came across an interesting problem I can't understand. The `test_execution.review_decision` still seems to be of a type <list>. However, when I tested this endpoint, it indeed showed `set` behaviour. Even if I passed the same argument twice or more, it still compressed it into one. I'm not sure what might be the cause of this.

Reply:
I guess it's fine if the endpoint just converted the request to a set